r/R/errorCodes.R

```diff
@@ -0,0 +1,6 @@
+errorCodes <- list(
```

Comment:
this same structure in python was called `errorCode`, now it is called `errorCodes`. Having sometimes s at the end and sometimes not will be very confusing. I suggest you call them both with `s` or both without `s`.

Reply:
changed